arrays/README.md

```diff
@@ -1,3 +1,6 @@
+
+[C-CPP.zip](https://github.com/MannyP31/DSA/files/6319513/C-CPP.zip)
+[evenodd.zip](https://github.com/MannyP31/DSA/files/6319516/evenodd.zip)
```

Comment:
Don't add the topics linking to your fork. add the path of the code file. Like-
[topic_name](c-or-cpp/file_name.c) and [topic_name](java/file_name.java)
Just add the file path of all your algos in their appropriate README.md file, it will link correctly to your file.

Reply:
Add these changes to the README.md file of arrays and sorting folders.

Reply:
yeah now it's done i guess.